pyro/infer/importance.py

```diff
@@ -79,3 +81,43 @@ def get_ESS(self):
             warnings.warn("The log_weights list is empty, effective sample size is zero.")
             ess = 0
         return ess
+
+
+def vectorized_importance_weights(model, guide, *args, **kwargs):
+
+    num_samples = kwargs.pop("num_samples", 1)
+    max_plate_nesting = kwargs.pop("max_plate_nesting", 7)
```

Comment:
nit: I think it might be safer to let this throw an exception if `max_plate_nesting` isn't provided than to assume a default value.